symbols/overlay11.yml

```
@@ -279,6 +279,14 @@ overlay11:
         Implements SPECIAL_PROC_0x16 (see ScriptSpecialProcessCall).
         
         r0: bool
+    - name: IsScreenFadeNotDone
+      address:
+        EU: 0x22EA8B8
+      description: |-
+        Used for example in the handler functions of the top screen types in ground mode
+        to check whether the top screen fade is complete or not.
+        
+        return: bool
```

Might be a good idea to specify exactly when `true` and `false` are returned, just to make sure there's no doubts. I've seen functions where the return value wasn't obvious.

Maybe

```suggestion
        return: True if the top screen is still fading, false if it's done fading.
```